### PR TITLE
App fix fonts

### LINKED ISSUE #15: Add custom fonts and correct font color
- The fonts to be used should be:
  - [Zen Maru Gothic](https://fonts.google.com/specimen/Zen+Maru+Gothic) for logos only. (Black)
  - [Zen Kaku Gothic New](https://fonts.google.com/specimen/Zen+Kaku+Gothic+New) should be used for texts. (Regular, Bold, Black).
- Font color must be **white**

### LINKED ISSUE #16: Missing written logo in Header with custom font

### PR DESCRIPTION
**Pull request by Fabio-Ricci**

### CODE REVIEW

**Comment by paulopacitti** [comment]
As noted in the Design Document, only the font weights below should be used. No worry about removing the additional weight formats, just make sure to use the right weight for each text :)

Screen Shot 2022-09-21 at 13 53 43

File: app/src/components/Header.tsx [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
-        <Text className="text-white">Settings</Text>
-        <Text className="font-bold text-xl text-white">Umbrella?</Text>
-        <Text className="text-white">Notifications</Text>
-      </View>
-    );
+  return (
+    <View className="items-center justify-between flex-row px-5 android:mt-4">
+      <Text className="text-white font-zenMaru">Settings</Text>
+      <Text className="font-bold text-xl text-white font-zenMaru">
+        Umbrella?
```
Comment (paulopacitti) [inline app/src/components/Header.tsx:9]: Logo is in lowercase format = "umbrella?"

File: app/src/screens/HomeScreen.tsx [hunk 2]
```diff
@@ -36,7 +36,9 @@ const HomeScreen: React.FC<PropsWithChildren<{}>> = () => {
       <View className="my-1">
         <Widget>
           <View>
-            <Text className="text-center">{item.sentence}</Text>
+            <Text className="text-center text-white font-zenKakuNew">
+              {item.sentence}
```
Comment (paulopacitti) [inline app/src/screens/HomeScreen.tsx:40]: This must be a "medium" weight format, like suggested in Figma

File: app/src/components/Header.tsx (hunk 1 above)
Comment (paulopacitti) [inline app/src/components/Header.tsx]: Logo is in lowercase format with "black" weight, like suggested on Figma = "umbrella?"

File: app/src/components/Header.tsx [hunk 3]
```diff
@@ -1,29 +1,16 @@
...
-    return (
-      <View className="items-center justify-between flex-row px-5">
-        <Text className="text-white">Settings</Text>
-        <Text className="font-bold text-xl text-white">Umbrella?</Text>
-        <Text className="text-white">Notifications</Text>
-      </View>
-    );
+  return (
+    <View className="items-center justify-between flex-row px-5 android:mt-4">
+      <Text className="text-white font-zenMaru">Settings</Text>
```
Comment (paulopacitti) [inline app/src/components/Header.tsx]: This should be "medium" or "bold", try both an choose one that you think is the best

**Review by paulopacitti — CHANGES_REQUESTED** [review]
Some minor issues that are a easy fix, just to make sure we have a good app to show tomorrow ;)

File: app/src/components/Header.tsx [hunk 4]
```diff
@@ -1,29 +1,16 @@
 import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-
-const styles = StyleSheet.create({
-  android: {
-    top: 35,
```
Comment (paulopacitti) [inline app/src/components/Header.tsx:6]: Don't remove this, it's necessary for android builds

**Review by paulopacitti — CHANGES_REQUESTED** [review]

**Comment by paulopacitti** [comment]
Make sure to merge the branch with `main` after the fixes, so we don't find any conflicts when merging this PR

**Comment by paulopacitti** [comment]
Good code!

File: app/src/components/Header.tsx (hunk 4 above)
Comment (Fabio-Ricci) [reply app/src/components/Header.tsx:6]: I've added the CSS style directly on the JSX via NativeWind CSS (android:top-9 - it's 36px), I think it's a better practice.

I we can change the 36px later if the amount is too much or too little

Comment (paulopacitti) [reply app/src/components/Header.tsx:6]: Deal! The StyleSheet uses density pixels, so it scale well for other devices. In the way that you explained, it seems that does the same effect. I'll approve and merge tomorrow

**Pull request merged**